pkg/config/gcp.go

```diff
@@ -2,7 +2,7 @@ package config
 
 // GCPConfig specifies the properties required to use GCP as the storage backend
 type GCPConfig struct {
-	ProjectID string `envconfig:"GOOGLE_CLOUD_PROJECT"`
-	Bucket    string `validate:"required" envconfig:"ATHENS_STORAGE_GCP_BUCKET"`
-	JSONKey   string `envconfig:"ATHENS_STORAGE_GCP_JSON_KEY"`
+	ProjectID string `envconfig:"GOOGLE_CLOUD_PROJECT" json:"projectID"`
```

Comment:
@fedepaol does this affect any of the config decoding we do? Guessing no because it's TOML but just want to check

Reply:
I assumed it doesn't, as the annotation is for json. Also, looking [at the lib we use](https://github.com/BurntSushi/toml), the annotation syntax is `toml:foo`.

Reply:
also also, if this was affecting decoding, tests would've failed